freqtrade/freqai/prediction_models/XGBoostRegressor.py

```diff
@@ -36,8 +36,8 @@ def fit(self, data_dictionary: dict, dk: FreqaiDataKitchen, **kwargs) -> Any:
             eval_set = None
             eval_weights = None
         else:
-            eval_set = [(data_dictionary["test_features"], data_dictionary["test_labels"]), (X, y)]
-            eval_weights = [data_dictionary["test_weights"], data_dictionary["train_weights"]]
+            eval_set = [(data_dictionary["test_features"], data_dictionary["test_labels"])]
```

Comment:
Breaking other people's logs without any rationale is not ok, remove this file from PR.